cypress/support/step_definitions/fireboltCalls.js

```diff
@@ -158,34 +158,36 @@ Given(/'(.+)' invokes the '(.+)' API to '(.+)'$/, async (appId, sdk, key) => {
 
         // Sending message to 3rd party app.
         cy.sendMessagetoApp(requestTopic, responseTopic, parsedIntent).then((result) => {
-          if (result === CONSTANTS.NO_RESPONSE) {
-            assert(false, CONSTANTS.NO_MATCHED_RESPONSE);
-          }
-          result = JSON.parse(result);
+          if (!Cypress.env('isRpcOnlyValidation')) {
+            if (result === CONSTANTS.NO_RESPONSE) {
+              assert(false, CONSTANTS.NO_MATCHED_RESPONSE);
+            }
+            result = JSON.parse(result);
 
-          // Create a deep copy to avoid reference mutation
-          const dataToBeCensored = _.cloneDeep(result.report.apiResponse);
+            // Create a deep copy to avoid reference mutation
+            const dataToBeCensored = _.cloneDeep(result.report.apiResponse);
 
-          // Call the 'censorData' command to hide sensitive data
-          cy.censorData(method, dataToBeCensored).then((maskedResult) => {
-            cy.log(`Response from ${appId}: ${JSON.stringify(maskedResult)}`);
-          });
+            // Call the 'censorData' command to hide sensitive data
+            cy.censorData(method, dataToBeCensored).then((maskedResult) => {
+              cy.log(`Response from ${appId}: ${JSON.stringify(maskedResult)}`);
+            });
 
-          // If method and params are not supported setting isScenarioExempted as true for further validation.
-          if (UTILS.isScenarioExempted(method, param)) {
-            Cypress.env(CONSTANTS.IS_SCENARIO_EXEMPTED, true);
-          }
+            // If method and params are not supported setting isScenarioExempted as true for further validation.
+            if (UTILS.isScenarioExempted(method, param)) {
+              Cypress.env(CONSTANTS.IS_SCENARIO_EXEMPTED, true);
+            }
 
-          // Creating object with method name, params and response etc and storing it in a global list for further validation.
-          const apiAppObject = new apiObject(
-            method,
-            param,
-            context,
-            result.report,
-            expected,
-            appId
-          );
-          UTILS.getEnvVariable(CONSTANTS.GLOBAL_API_OBJECT_LIST).push(apiAppObject);
+            // Creating object with method name, params and response etc and storing it in a global list for further validation.
+            const apiAppObject = new apiObject(
+              method,
+              param,
+              context,
+              result.report,
+              expected,
+              appId
+            );
+            UTILS.getEnvVariable(CONSTANTS.GLOBAL_API_OBJECT_LIST).push(apiAppObject);
+          }
```

Comment:
Can you add a log after the api call in case of rpcOnlyValidation 
![image](https://github.com/rdkcentral/firebolt-certification-suite/assets/148241508/631ab04d-0c06-44f4-8e68-1f573828d8de)

[6/17/2024 2:45:14 PM]["Call from comcast.test.firecert, method: keyboard.email params: {\"type\":\"signIn\",\"message\":\"Enter your email to sign into this app\"}"]
[6/17/2024 2:45:14 PM]["Not waiting for 'keyboard.email' response. It will be fetched in the subsequent steps and validated when rpc-only methods are invoked"}"]

Reply:
Added log as follows :
      if (Cypress.env('isRpcOnlyValidation')) {
        cy.log(`${method} response will be retrieved in subsequent steps and validated when the rpc-only methods are invoked. Proceeding to the next step.`);
      }